### ISSUE TITLE
Manual test types integration

### ISSUE THREAD

**Issue by jbprat** [jan-molak/serenity-js#79]
Hi,

I'm trying to integrate manual testing using the @manual tag before my scenario, following these guidelines: https://thucydides-webtests.com/category/tips-and-tricks/
```
  @manual
  Scenario: Load the Header Component 2 in the Test Interface page.
    Verify the logo is displayed.
    Verify the brandname is displayed.
    Verify what text is displayed as brandname.
```

But, nothing shows in my serenity report in the manual test types line. What is the way to add items to this line?

**Comment by jbprat** [comment]
Got my answer from here: https://github.com/jan-molak/serenity-js/issues/67, it's not supported yet.
When do you think will it be implemented?

**Comment by jan-molak** [comment]
It's implemented in [@serenity-js/core 1.5.2](https://github.com/jan-molak/serenity-js/commit/babc5878ebc6e9be4686df1419c52c49298b32f7).

Best,
Jan

**Comment by jbprat** [comment]
Thanks!

Envoyé depuis ProtonMail mobile

-------- Message d'origine --------
